README.md

```diff
@@ -1 +1,91 @@
-Envoy v2 gRPC APIs
+# Envoy v2 gRPC APIs
+
+## Goals
+
+This repository contains the draft v2 gRPC
+[Envoy](https://github.com/lyft/envoy/) APIs. Envoy today has a number of JSON
+REST APIs through which it may discover and have updated its runtime
+configuration from some management server. These are:
+
+* [Cluster Discovery Service (CDS)](https://lyft.github.io/envoy/docs/configuration/cluster_manager/cds.html)
+* [Rate Limit Service (RLS)](https://lyft.github.io/envoy/docs/configuration/overview/rate_limit.html)
+* [Route Discovery Service (RDS)](https://lyft.github.io/envoy/docs/configuration/http_conn_man/rds.html)
+* [Service Discovery Service (SDS)](https://lyft.github.io/envoy/docs/configuration/cluster_manager/sds_api.html)
+
+Version 2 of the Envoy API will evolve existing APIs and introduce new APIs to:
+
+* Allow for more advanced load balancing through load and resource utilization reporting to management servers.
+* Improve N^2 health check scalability issues by optionally offloading health checking to other Envoy instances.
+* Support Envoy deployment in edge, sidecar and middle proxy deployment models via changes to the listener model and CDS/SDS APIs.
+* Allow streaming updates from the management server on change, instead of polling APIs from Envoy. gRPC APIs will be supported
+  alongside JSON REST APIs to provide for this.
+* Ensure all Envoy runtime configuration is dynamically discoverable via API
```

Comment:
there still will be bootstrap config that Envoy will need, right? It would be nice to at least mention it here. We also need to figure out what goes there.